src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -535,6 +535,7 @@ def load_arguments(self, _):
         c.argument('skip_resource_name_params', action='store_true')
         c.argument('skip_all_params', action='store_true')
         c.argument('resource_ids', nargs='+', options_list='--resource-ids')
+        c.argument('export_format', options_list=['--export-format'], help="The format of the exported template. Allowed values: 'json', 'arm', 'bicep'. Defaults to 'json'.")
```

Comment:
We don't need to declare the default option name 
```suggestion
        c.argument('export_format', help="The format of the exported template. Allowed values: 'json', 'arm', 'bicep'. Defaults to 'json'.")
```